### PR TITLE
Modified CONTRIBUTING.md

### LINKED ISSUE #8: Revise CONTRIBUTING.md
https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md is a good example

### PR DESCRIPTION
**Pull request by elevran**
Fixes #8

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,33 +1,72 @@
-Amalgam8 service discovery registry is actively developed by [IBM](http://www.ibm.com) as an open source project - 
-we encourage anyone to contribute through pull requests, issue reporting, feature requests, etc.
+### Welcome
 
-If you have a change or new feature in mind, please [suggest it by creating an issue](https://github.com/amalgam8/registry/issues) 
-and tag it with "feature proposal"
+We welcome contributions to the Amalgam8 Project in many forms, and there's always plenty to do!
 
-Please ensure that you are able to certify pull requests based on the below statement:
+First things first, please review the project's [Code of Conduct](http://contributor-covenant.org/version/1/4/) before participating. It is important that we keep things civil.
```
Comment (christo4ferris) [inline CONTRIBUTING.md]: This seems like a decent enough coc but, it recommends that projects not simply reference it, but also modify to specify cobtact for reporting incidents and also to document a process for handling enforcement. There is a npm approach to adding it to projects, suggest you use that. Also would strongly recommend attribution.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,33 +1,72 @@
...
 
-Developer's Certificate of Origin 1.1
+### Reporting bugs
+If you are a user and you find a bug, please submit an [issue](https://github.com/amalgam8/registry/issues). Please try to provide sufficient information for someone else to reproduce the issue.
```
Comment (christo4ferris) [inline CONTRIBUTING.md]: if you intend to link this to other projects, then you'll want to deal with relative location for issues, no?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,33 +1,72 @@
...
+First things first, please review the project's [Code of Conduct](http://contributor-covenant.org/version/1/4/) before participating. It is important that we keep things civil.
 
-Developer's Certificate of Origin 1.1
+### Reporting bugs
+If you are a user and you find a bug, please submit an [issue](https://github.com/amalgam8/registry/issues). Please try to provide sufficient information for someone else to reproduce the issue. 
+One of the project's maintainers should respond to your issue within 24 hours. If not, please bump the issue and request that it be reviewed.
 
-       By making a contribution to this project, I certify that:
+### Fixing issues and working stories
+Review the [issues list](https://github.com/amalgam8/registry/issues) and find something that interests you. You could also check the ["help wanted"](https://github.com/amalgam8/registry/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) list.
```
Comment (christo4ferris) [inline CONTRIBUTING.md]: same comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (elevran) [inline CONTRIBUTING.md]: Makes sense. Should probably have top level repository to hold project wide files, such as code of conduct, issue templates, etc. 
These would need to be repository neutral (i.e., not link specific paths).

File: CONTRIBUTING.md (hunk 2 above)
Comment (elevran) [inline CONTRIBUTING.md]: Yes, need to consider if it makes sense to link (in which case this needs to be repository agnostic) or duplicate. See above as well.

**Pull request closed without merging**